packages/shared/src/authz/domain/authErrors.ts

```diff
@@ -90,6 +90,16 @@ export const WorkspaceLimitsReachedError = defineAuthError<
   message: 'Workspace limits have been reached'
 })
 
+export const WorkspaceProjectMoveInvalidError = defineAuthError({
+  code: 'WorkspaceProjectMoveInvalid',
+  message: 'Projects already in a workspace cannot be moved to another workspace.'
+})
+
+export const WorkspaceRequiredError = defineAuthError({
+  code: 'WorkspaceRequired',
+  message: 'You must be in a workspace to perform this action.'
```

Comment:
I'm confused by the message. "You" the user must be in a workspace? Or a project must be in a workspace?